paymentsheet/src/main/java/com/stripe/android/paymentsheet/navigation/PaymentSheetScreen.kt

```diff
@@ -304,7 +304,7 @@ internal sealed interface PaymentSheetScreen {
 
         @Composable
         override fun Content(modifier: Modifier) {
-            PaymentMethodVerticalLayoutUI(interactor)
+            PaymentMethodVerticalLayoutUI(interactor, modifier)
```

Comment:
we need to pass this modifier through, because `PaymentSheetScreen` sets bottom padding for the content that is necessary to ensure there's padding between the mandate and this content